client/src/components/layouts/rtl-layout.css

```diff
@@ -100,6 +100,28 @@ universal-nav-bar
 and menu secion
 */
 
+@media (max-width: 768px) {
+  [dir='rtl'] .fcc_searchBar .ais-Hits {
+    left: auto;
+    right: 15px;
+  }
+}
+
+[dir='rtl'] .ais-SearchBox-reset {
+  right: auto;
+  left: 0;
+}
+
+[dir='rtl'] .universal-nav-right .ais-SearchBox-reset {
+  left: 291px;
```

Comment:
This is quite a magical number. I think we need to find a solution that does not involve hard-coding values.

Reply:
Yeab I have it, I am planning to open two issues after this.

But the short version is to change to grid and use pseudo class to have the input border

Reply:
Opened issue for this https://github.com/freeCodeCamp/freeCodeCamp/issues/48912, and if the camper is having issue with their PR, I will try to guide them instead of assuming, it won't deal well with conflicts